packages/@vue/cli/bin/vue.js

```diff
@@ -57,6 +57,10 @@ program
   .option('-b, --bare', 'Scaffold project without beginner instructions')
   .action((name, cmd) => {
     const options = cleanArgs(cmd)
+    if(program.args.length > 1){
+      program.outputHelp()
+      console.log(chalk.red.bold('\n Kindly provide only one argument as the appname'))
```

Comment:
We don't throw an error or anything (rightfully so), we continue processing, so asking the user to only provide one argument when he can't (we continue processing, right?), I would rather phrase it like this:

> Info: You provided more than one argument. The first one will be used as the app's name, the rest are ignored.

Reply:
Rephrased the comment as you said and removed logging help.